packages/react/src/Card/Card.tsx

```diff
@@ -124,6 +127,19 @@ const CardRoot = forwardRef<HTMLAnchorElement, CardProps>(
   }
 )
 
+type CardImageProps = BaseProps<HTMLImageElement> & {
+  src: ImageProps['src']
+  alt: ImageProps['alt']
+}
```

Comment:
@josepmartins I think props like `aspectRatio` won't work unless you create an intersection with `ImageProps`. 

I think this would need to change to:

```suggestion
type CardImageProps = ImageProps
```

`src` and `alt` are already required in that component, and ImageProps also extends HTMLImageElement already.

Reply:
good point 👍 done